src/test/java/io/antmedia/test/settings/ServerSettingsTest.java

```diff
@@ -32,7 +32,7 @@ public void testNativeLogLevel() {
 		ServerSettings settings = new ServerSettings();
 		
 		assertEquals(Logging.Severity.LS_WARNING, settings.getWebRTCLogLevel());
-		assertEquals(ServerSettings.LOG_LEVEL_WARN, settings.getNativeLogLevel());
+		assertEquals(ServerSettings.LOG_LEVEL_ERROR, settings.getNativeLogLevel());
```

Comment:
This test code is checking the default value. It's good to fix it.

On the other hand, there is no check if `setNativeLogLevel` is called in  `setApplicationContext` method which is part of the fix. So we may encounter same problem again and we may need to spend time again for our previous fixes. 

Remember the manifesto item - https://antmedia.io/about-us/
- Solve problems permanently. Don’t waste time addressing the same issue multiple times.

Reply:
Got it , I will update the PR

Reply:
I've added following lines for the test code

https://github.com/ant-media/Ant-Media-Server/pull/5772/files#diff-98b52309ed36b413f4fd32a8be8fcf05fcd787de4f3d5c6d1d6429f43b9ed571R94